.github/workflows/synchronizing.yaml

```diff
@@ -96,8 +96,8 @@ jobs:
 
       - name: apply changes to the files
         run: |
-          if echo "${{ github.event.head_commit.message }}" | grep -q "#"; then
-            branch_name=$(echo "${{ github.event.head_commit.message }}" | grep -o '[0-9]\+')
+          if echo "${{ github.event.head_commit.message }}" | grep -q '\(#\d*\)'; then
```

Comment:
Whats an example of the undesirable jira merge-id that this pattern matches?
Suggest adding a comment to make it clear for future.

Reply:
[CDO-123]

the merge ID will be something like (#23), so i'm just matching for the mergeID instead

Reply:
Regex should have # and at least 1 number?
Also '\d' is PCRE/exgended regex. Should use grep -e or -P.
Highly recommend a comment. Code intent is not self-evident.